sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputation.java

```diff
@@ -1,53 +1,266 @@
-/**
+/*
  * Copyright (c) 2018, RTE (http://www.rte-france.com)
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 package com.powsybl.sensitivity;
 
+import com.google.common.base.Suppliers;
+import com.powsybl.commons.PowsyblException;
 import com.powsybl.commons.Versionable;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.util.ServiceLoaderCache;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.computation.DefaultComputationManagerConfig;
 import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.iidm.network.Network;
 
+import java.util.List;
+import java.util.Objects;
 import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
 
 /**
- * Sensitivity computation interface
+ * Sensitivity computation main API. It is a utility class (so with only static methods) used as an entry point for running
+ * a sensitivity computation allowing to choose either a specific implementation or just to rely on default one.
  *
- * <p>
- *     Sensitivity computation is used to assess the impact of a small modification
- *     of a network variables on the value of network functions.
- *     This computation can be assimilated to a partial derivative computed on a given
- *     network state and on that state modified based on a list of contingencies, if specified.
- * </p>
- * <p>
- *     PTDFs used in Flowbased methodology for example are sensitivity computation
- *     results. The sensitivity variables are the GSK shift and the sensitivity function
- *     are the monitored lines/transformers flows.
- * </p>
- * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
-public interface SensitivityComputation extends Versionable {
+public final class SensitivityComputation {
```

Comment:
Maybe since we're breaking the API this would be an opportunity to rename SensitivityComputation into SensitivityAnalysis everywhere, since this is the standard name for it in English it seems.

Reply:
Indeed, I did rename all the files and comments.